.node-version

```diff
@@ -0,0 +1 @@
+14
```

Comment:
I dunno anything about nvm, though I'm curious why these tools don't inspect package.json's engine info for this?

Seems like a reasonable change to me though!

Reply:
Agreed, you would think it would be smarter!

Reply:
Ahh, [open source](https://github.com/nvm-sh/nvm/issues/651) 😆